src/CloneLeeroy/Program.cs

```diff
@@ -58,7 +58,7 @@ public static async Task<int> Main(string[] args)
 				catch (LeeroyException ex)
 				{
 					using (SetColor(ConsoleColor.Red))
-						Console.Error.WriteLine(ex.Message);
+						await Console.Error.WriteLineAsync(ex.Message);
```

Comment:
This is an example of an analyzer recommending a change that seems unnecessary and possibly unwise. Unless there is some advantage to async console I/O? I wish they'd omitted console interactions from the analyzer.

Reply:
Ignoring it in the csproj seems fine with me. Or with a pragma at the top of this file that mentions the console, e.g. `// console sync I/O`